src/User32.Desktop/User32.cs

```diff
@@ -109,6 +128,12 @@ public static partial class User32
         [DllImport(nameof(User32))]
         public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
 
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, StringBuilder lParam);
+
+        [DllImport(nameof(User32))]
+        public static extern int SendMessage(IntPtr hWnd, int wMsg, uint wParam, int lParam);
```

Comment:
> (UTF8 and UTF16)

I suspect you meant ANSI and UTF-16. But aren't strings treated as auto by messages ?

Searching fast on the web it seem that when `SendMessageA` is called the messages expect an ANSI string and when `SendMessageW` is called an UTF-16 string is expected.

The only thing is that even if it doesn't explicitly use strings we need to specify `CharSet.Auto` on `SendMessage` to be able to call the correct A/W variant.